src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -16,24 +16,84 @@ public void GetAccessControl_DirectoryInfo_InvalidArguments()
         }
 
         [Fact]
+        public void GetAccessControl_DirectoryInfo_ReturnsValidObject()
+        {
+            using (var directory = new TempDirectory())
+            {
+                DirectoryInfo directoryInfo = new DirectoryInfo(directory.Path);
+
+                DirectorySecurity directorySecurity = FileSystemAclExtensions.GetAccessControl(directoryInfo);
```

Comment:
I think off the object is more idomatic and would be a good idea. 

nearly there!